test/MatrixFields/flat_spaces.jl

```diff
@@ -0,0 +1,66 @@
+include("matrix_field_test_utils.jl")
+import ClimaCore.MatrixFields: @name, ⋅
+function point_space(::Type{FT}) where {FT}
+    comms_ctx = ClimaComms.SingletonCommsContext(comms_device)
+    coord = Geometry.ZPoint(FT(0))
+    return Spaces.PointSpace(comms_ctx, coord)
+end
+
+
+function se2d_space(::Type{FT}) where {FT}
```

Comment:
There's already a function for this in `TestUtilities`: `SpectralElementSpace2D`

Reply:
thanks!